### ISSUE TITLE
Feature Request: options to sync/desync an application using CLI

### ISSUE THREAD

**Issue by easypickings** [lra/mackup#1892]
I find it quite indirect when trying to desync a single synced application.
I have to manually copy back the file(s) and modify  the config.
It would be convenient by adding CLI options like `mackup sync <application>` and `mackup desync <application>` to automatically get the works done.

**Comment by joshmedeski** [comment]
I agree this would be helpful. @lra, do you have some time to work on this soon? I might be able try it out in the next quarter.
